src/Microsoft.VisualStudio.Editors/ResourceEditor/ResourceEditorView.vb

```diff
@@ -3268,7 +3268,7 @@ Namespace Microsoft.VisualStudio.Editors.ResourceEditor
                 If Not String.IsNullOrEmpty(fileExtension) Then
                     Dim isSafe As Boolean = False
 
-                    ' first of all, we check the white list for the resource type. If the extension matches one of them, we won't pop up a warning dialog.
+                    ' first of all, we check the safe list for the resource type. If the extension matches one of them, we won't pop up a warning dialog.
```

Comment:
What about 'allow list'? https://en.wikipedia.org/wiki/Whitelisting#Controversy

Reply:
I would ordinarily call it that, but it's referred to as a safe list in subsequent code so I feel that's a more consistent name here.